explainer.md

```diff
@@ -0,0 +1,386 @@
+
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
+<!-- DON'T EDIT THIS SECTION, INSTEAD RE-RUN doctoc TO UPDATE -->
+## Contents
+
+- [Motivation](#motivation)
+  - [Goals](#goals)
+  - [Non-Goals](#non-goals)
+- [Proposed Solution: Secure Payment Confirmation](#proposed-solution-secure-payment-confirmation)
+  - [Proposed APIs](#proposed-apis)
+    - [Creating a credential](#creating-a-credential)
+      - [Creating a SPC credential in a cross-origin iframe](#creating-a-spc-credential-in-a-cross-origin-iframe)
+    - [Authenticating a payment](#authenticating-a-payment)
+- [Other Considerations](#other-considerations)
+  - [Initial Experimentation with Stripe](#initial-experimentation-with-stripe)
+  - [Are SPC credentials identical to WebAuthn credentials?](#are-spc-credentials-identical-to-webauthn-credentials)
+  - [Why use the PaymentRequest API](#why-use-the-paymentrequest-api)
+- [Alternatives Considered](#alternatives-considered)
+  - [Traditional WebAuthn](#traditional-webauthn)
+  - [Delegated Authentication](#delegated-authentication)
+- [Security Considerations](#security-considerations)
+  - [Enrollment in cross-origin iframes](#enrollment-in-cross-origin-iframes)
+  - [Cross-origin authentication ceremony](#cross-origin-authentication-ceremony)
+  - [Merchant-supplied data](#merchant-supplied-data)
+- [Privacy Considerations](#privacy-considerations)
+  - [Probing](#probing)
+  - [Credential-sharing](#credential-sharing)
+
+<!-- END doctoc generated TOC please keep comment here to allow auto update -->
+
+## Motivation
+
+Online payments are usually a 3 party interaction:
+
+* The Merchant (e.g. an online web store)
+* The Customer (e.g. the user buying from an online web store)
+* The Account Provider (e.g. the bank that issued the payment instrument being used)
+
+> **NOTE**: It is not uncommon for there to be a fourth party: the Payment
+> Service Provider, whom the Merchant may delegate the task of accepting the
+> payment method to. This is usually done by the Merchant either redirecting the
+> User to the Payment Service Provider's website, or by embedding the Payment
+> Service Provider in an iframe. Any following mention of the term Merchant
+> should be taken to mean either a Merchant or a Payment Service Provider;
+> whichever is handling payment.
+
+Traditionally, the Customer only interacts with the Merchant, and the Merchant
+communicates with the Account Provider via a back-channel protocol. However,
+Account Providers are increasingly looking to authenticate Customers during
+online payments. This is done for both regulatory reasons (e.g. [SCA] in the
+European Union) or for fraud prevention reasons - either fradulent Customers or
+fradulent Merchants.
+
+Existing methods of authenticating a Customer during an online payment are
+either high friction (e.g. embedding challenge iframes from the Account
+Provider) or have poor user privacy (e.g. data-collection or tracking the user
```

Comment:
Suggest "high friction (e.g. embedding challenge iframes from the Account Provider or redirecting to the Account Provider site or app)"